src/main/java/io/orangebeard/listener/OrangebeardTestSystemListener.java

```diff
@@ -1,141 +1,159 @@
 package io.orangebeard.listener;
 
-import com.epam.reportportal.service.Launch;
-import com.epam.reportportal.service.ReportPortal;
-import com.epam.ta.reportportal.ws.model.FinishExecutionRQ;
-import com.epam.ta.reportportal.ws.model.FinishTestItemRQ;
-import com.epam.ta.reportportal.ws.model.StartTestItemRQ;
-import com.epam.ta.reportportal.ws.model.attribute.ItemAttributesRQ;
-import com.epam.ta.reportportal.ws.model.launch.Mode;
-import com.epam.ta.reportportal.ws.model.launch.StartLaunchRQ;
-import com.google.api.client.http.*;
-import com.google.api.client.http.javanet.NetHttpTransport;
 import fitnesse.testrunner.WikiTestPage;
-import fitnesse.testsystems.*;
+import fitnesse.testsystems.Assertion;
+import fitnesse.testsystems.ExceptionResult;
+import fitnesse.testsystems.TestPage;
+import fitnesse.testsystems.TestResult;
+import fitnesse.testsystems.TestSummary;
+import fitnesse.testsystems.TestSystem;
+import fitnesse.testsystems.TestSystemListener;
 import fitnesse.wiki.PageData;
 import fitnesse.wiki.WikiPageProperty;
+import io.orangebeard.client.OrangebeardClient;
```

Comment:
Misschien ook ff de .editorconfig overnemen uit de java client, dan worden de imports in een schitterende volgorde gezet oa...